resources/assets/less/bem/tooltip-beatmap.less

```diff
@@ -0,0 +1,74 @@
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+.tooltip-beatmap {
+  min-width: 110px;
+  max-width: none;
+
+  height: 35px;
+
+  .default-border-radius;
+
+  background-color: #333333;
+  border: 0 solid transparent;
+
+  .qtip-content {
+    width: 100%;
+    height: 100%;
+
+    .center-content;
```

Comment:
the only thing used from this function is `display`. The rest are overridden by the two following lines.

Reply:
Right. Though now I look again, it can be slightly simplified:

```diff
diff --git a/resources/assets/less/bem/tooltip-beatmap.less b/resources/assets/less/bem/tooltip-beatmap.less
index 4e55e92e9..4a1d70747 100644
--- a/resources/assets/less/bem/tooltip-beatmap.less
+++ b/resources/assets/less/bem/tooltip-beatmap.less
@@ -20,32 +20,22 @@
   min-width: 110px;
   max-width: none;

-  height: 35px;
+  .default-border-radius();

-  font-size: 11px;
-
-  .default-border-radius;
-
-  background-color: #333333;
+  background-color: #333;
   border: 0 solid transparent;

   .qtip-content {
     width: 100%;
     height: 100%;
-
-    .center-content;
-    flex-direction: column;
-    justify-content: space-between;
-
-    padding: 4px 10px;
+    padding: 2px 10px;
   }

-  &__title {
-    font-weight: 700;
-    color: #ffffff;
-  }
+  &__text {
+    text-align: center;
+    font-size: 11px;
+    margin: 4px 0;

-  &__stars {
     &--easy {
       color: @green;
     }
@@ -69,5 +59,10 @@
     &--expert-plus {
       color: #000000;
     }
+
+    &--title {
+      font-weight: 700;
+      color: #fff;
+    }
   }
 }
```